3scale/porta
# THREESCALE-8735: Client secret in clear text in the API Manager UI (RH SSO Integration) Part 1

**What this PR does / why we need it**:
Currently, our admin SSO Integrations display the client secret when checking an integration as long as when editing or creating a new one.

**Which issue(s) this PR fixes** 
[THREESCALE-8735](https://issues.redhat.com/browse/THREESCALE-8735). This first part deals with the show template, hiding the client secret. 

**Before**
![Captura de pantalla 2024-07-01 a las 13 30 32](https://github.com/3scale/porta/assets/72191439/bdbb54e2-8441-4677-82fb-621766207de1)

**After**
![Captura de pantalla 2024-07-03 a las 13 58 29](https://github.com/3scale/porta/assets/72191439/dd86341a-d42c-4401-a2a3-ec8926129144)

**Verification steps** 
1. Go to Account Settings > Users > SSO Integrations
2. If you don't have an SSO Integration, create one
3. Confirm `Client Secret` is not present
